.teamcity/common/BuildCache.kt

```diff
@@ -35,7 +34,7 @@ val builtInRemoteBuildCacheNode = RemoteBuildCache("%gradle.cache.remote.url%")
 
 object NoBuildCache : BuildCache {
     override fun gradleParameters(os: Os): List<String> {
-        return emptyList()
+        return listOf("--no-build-cache")
```

Comment:
I think `NoBuildCache` means no remote build cache. The local build cache should still be active as it is now, right?

Reply:
`disablesBuildCache` is not used any more, will remove this, it must be a legacy leftover.

Reply:
And I think `NoBuildCache` means no build cache at all, local and remote, for example, for `Trigger` build.

Reply:
We used it to compare with and without build cache runs. You can remove the class now, we can re-add it if we need it.